Gems/EMotionFX/Code/Source/Integration/Editor/Components/EditorActorComponent.cpp

```diff
@@ -139,6 +140,9 @@ namespace EMotionFX
                         ->Attribute(AZ::Edit::Attributes::ChangeNotify, &EditorActorComponent::OnSkinningMethodChanged)
                         ->EnumAttribute(SkinningMethod::DualQuat, "Dual quat skinning")
                         ->EnumAttribute(SkinningMethod::Linear, "Linear skinning")
+                        ->DataElement(AZ::Edit::UIHandlers::CheckBox, &EditorActorComponent::m_excludeFromReflectionCubeMaps, "Exclude from reflection cubemaps", "Actor will not be visible in baked reflection probe cubemaps")
```

Comment:
Do you have a use-case where you do want to enable actors in the reflection probes, or where you want to enable/disable via the MeshComponentBus? If not, then I'd say let's keep it simple and just always disable actors in reflection probes via https://github.com/o3de/o3de/pull/14108. We can always revive this change if the need arises.

If there is a use case, then I think we should ping sig-ui-ux for suggestions on where to put this attribute. I imagine we want to add a new group for render settings, put the group at the bottom of all the Actor component attributes, and have it collapsed by default, then put this setting in that group. It's something that most people will never touch, so we probably don't want it right at the surface of the UI. And with a render settings group, we can then add other mesh component settings like 'Use Forward Pass IBL Specular', or 'Use raytracing' (once we support that for skinned meshes).

Reply:
The only use case I can think of is when you use an actor for something that has a subtle or inactive animation like vegetation - not sure how common it is but we were going to use animated trees in the robotics demo to simulate wind in a pinch.

Reply:
Little bit like trees, we have a case where we could want to enable reflection probes on actors, entities like an audience.
In our case, we have an audience, so a group of many stuck entities together which can have a great influence in reflection probes, but in same time they have animations (to have a little bit of life) and so are actors.
I think this case works in both case precomputed and dynamic reflection probes, because the audience position doesn't change at runtime (for us), so can be taken into account for probes computation in editor without creating too big inconsistency in game.
Naturally dynamic case is more complex and will depend on what designers and developers want.
For example, in case of a First Person game (like exploration game) we could want to exclude actors for probes to save performances, but at the same time keep them "on" for the player character to have a good effect when the player come near a reflective surface.
Or more original, a game with vampires (no reflection) and humans (reflection) ^^.

Reply:
So I checked in #14108, so that Actors are by default hidden now, however the question of adding a control to make it optional for Actors remains.

Personally, I've come around to being ok with adding this control, but agree with @amzn-tommy that we should bury it in a collapsed "Advanced" section or something similar, and also the "Exclude from cube maps" control would default to on for Actors.

Pinging the ux sig for recommendations. @o3de/sig-ui-ux

Reply:
Another thing to keep in mind, if the UX for the Actor Component changes, then the docs will need to be updated as well - https://www.o3de.org/docs/user-guide/components/reference/animation/actor/

Reply:
Trees and crowds seem like plausible use cases for now (given a lack of built in vegetation bending for static meshes or bespoke crowd animation system), so adding a control to the UI makes sense to me. It just shouldn't be front and center.

Reply:
Looks like this is already under a group. Apologies for missing that the first time around. There is no indentation in the code where the group starts, so it was not obvious.